language-support/qute/qute.tmLanguage.json

```diff
@@ -198,15 +272,19 @@
         }
       },
       "patterns": [
+        {
```

Comment:
Remove thoses matches, user tag doesn't allow those operator, it is just for #if

Reply:
This isn't under the user tag match, this is under expression

Reply:
My bad you are right. Sorry.

But matches misses some operators and keywords.